<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chosecun</class>
 <widget class="QWidget" name="chosecun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 63 48pt &quot;Bahnschrift SemiBold SemiConden&quot;;</string>
     </property>
     <property name="text">
      <string>选村长</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="pushButton_3">
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>开始发言</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QPushButton" name="pushButton_4">
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;Agency FB&quot;;
font: 20pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>进入白天</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTextBrowser" name="textBrowser"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 25 22pt &quot;Bahnschrift Light&quot;;</string>
     </property>
     <property name="text">
      <string>请投票</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" rowspan="2">
    <widget class="QLineEdit" name="lineEdit">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>投票</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_5">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>投票结果</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QPushButton" name="pushButton">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
